lib/connection/commands.js

```diff
@@ -53,6 +53,9 @@ var Query = function(bson, ns, query, options) {
   this.returnFieldSelector = options.returnFieldSelector || null;
   this.requestId = Query.getRequestId();
 
+  // special case for pre-3.2 find commands, delete ASAP
+  this.pre32Limit = options.pre32Limit || null;
```

Comment:
Is there a reason we are using `null` here, and not `undefined`?

Reply:
I think simply for the sake of defining it with a value. I always feel weird assigning undefined to a property, as the intent would be to indicate it doesn't exist. Perhaps a better idea here is to do:
```
if (options.pre32Limit) this.pre32Limit = options.pre32Limit;
```

Reply:
Ok. In general, I'd like to not use `null` as it causes a lot of problems in JS. We could just do:

```js
this.pre32Limit = options.pre32Limit;
```

Also, if someone passes in `{ pre32Limit: 0 }`, the current behavior will set `this.pre32Limit` to `null`, not `0`. Do we want that behavior?

Reply:
```
this.pre32Limit = options.pre32Limit;
```

sounds good to me. setting it to null is fine.